extensions/extension_mgr.go

```diff
@@ -38,15 +38,15 @@ func init() {
 		if err != nil {
 			panic(err)
 		}
-		defer f.Close()
+		_ = f.Close()
```

Comment:
Sorry, I thought I moved it to the end of the loop. 
It was throwing a warning `Possible resource leak, 'defer' is called in the 'for' loop`

Reply:
yea, technically it's not *the best* idea to put the defer here because it won't be called until the end of the function itself. so moving it to the end of the loop is fine.